File: app/src/main/java/org/tianjyan/pwd/model/PasswordGroup.java
package org.tianjyan.pwd.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class PasswordGroup extends RealmObject {
    @PrimaryKey
    private String groupName;

    public String getGroupName() {
        return groupName;
    }

    public void setGroupName(String groupName) {
        this.groupName = groupName;
    }

    @Override
    public String toString() {
        return "PasswordGroup [groupName=" + groupName + "]";
    }
}

File: app/src/main/kotlin/org/tianjyan/pwd/model/PasswordGroup.kt
package org.tianjyan.pwd.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class PasswordGroup : RealmObject() {
    @PrimaryKey
    var groupName: String? = null

    override fun toString(): String {
        return "PasswordGroup [groupName=$groupName]"
    }
}